Added watch to glossary (#47067)
kubernetes/website: content/en/docs/reference/glossary/watch.md

---
title: Watch
id: watch
date: 2024-07-02
full_link: /docs/reference/using-api/api-concepts/#api-verbs
short_description: >
  A verb that is used to track changes to an object in Kubernetes as a stream.

aka:
tags:

> - core-object

Reviewer: ```suggestion
```
Watches are verbs, not objects.

Author: Ah! yes . thanks for the correction.

- fundamental
---
A verb that is used to track changes to an object in Kubernetes as a stream. It is used for the excellent detection of changes.

<!--more-->

> A verb that is used to track changes to an object in Kubernetes as a stream. It is used for the excellent detection of changes when this request is made to Kubernetes API or when HTTP GET request is made with watch query.

Reviewer: (optionally)
```suggestion
A verb that is used to track changes to an object in Kubernetes as a stream. Watches allow
efficient detection of changes; for example, a
{{< glossary_tooltip term_id="controller" text="controller">}} that needs to know whenever a
ConfigMap has changed can use a watch rather than polling.
```

Author: I guess an example will be beneficial. Thanks , I'll incorporate the changes.

---
title: Watch
id: watch
date: 2024-07-02
full_link: /docs/reference/using-api/api-concepts/#api-verbs
short_description: >
  A verb that is used to track changes to an object in Kubernetes as a stream.

aka:
tags:
- API verb
- fundamental
---

> A verb that is used to track changes to an object in Kubernetes as a stream. It is used for the excellent detection of changes.

Reviewer: (nit)
```suggestion
A verb that is used to track changes to an object in Kubernetes as a stream.
It is used for the efficient detection of changes.
```

Author: cool. thanks for the reviews.
